spec/resources_spec.rb

```diff
@@ -1,8 +1,21 @@
 require 'spec_helper'
 
 describe CfnDsl::ResourceDefinition do
-  subject { CfnDsl::CloudFormationTemplate.new.AutoScalingGroup(:web_servers) }
+  subject { CfnDsl::CloudFormationTemplate.new.EC2_Instance(:single_server) }
+  context '#all_refs' do
```

Comment:
Not sure this context name is really very descriptive of what we're checking here, unless I'm missing something?

I'm going to merge this though as it's fixing a blocking problem and we can re-visit naming in the specs in a subsequent PR.